CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added an optional prop `onPopOverClick` to pass a callback function to the `PopOver` UI component. This callback will be called when the `PopOver` UI component is clicked.
 - Added `--no-fail-on-empty-changeset` flag in deploy script to not fail for empty changeset.
 - Add `WithTooltip` docs, warning log if no container is found, and some additional flexibility.
+- Add support for optional keys to pass extra data in the `AttendeeResponse`.
```

Comment:
You need to add an [Unreleased] section and then add this to the new [Added] section up at the top.

Reply:
Updated the CHANGELOG.